Fix benchmarks build failure

majetideepak opened the pull request:
duckdb source includes a copy of re2 as a third-party dependency. However, it defines the re2 headers
 as public which seem to conflict with the system re2 headers.
The workaround is to not bundle duckdb.

mbasmanova reviewed (approved):
@majetideepak Deepak, thank you for the fix. Would you update the PR description to explain what is the problem and how this change is fixing it as it is not obvious?

majetideepak commented:
@mbasmanova I want to validate another fix first https://github.com/facebookincubator/velox/pull/10161.
I removed the ready-to-merge label. I updated the description.
